pkg/agent/agent.go

```diff
@@ -184,6 +191,10 @@ func (a *Agent) newEndpoints(ctx context.Context, cat catalog.Catalog, tel telem
 }
 
 func (a *Agent) bundleCachePath() string {
+	return path.Join(a.c.DataDir, "bundle.proto")
```

Comment:
I think the only thing we use the cached bundle for is re-connecting to the server after a restart? If so, I don't think we strictly need to update this since the only extra thing we are getting is jwt keys which aren't needed for agent -> server communication

Reply:
If the agent is able to load its SVID off disk, it doesn't re-attest and the attestor returns the bundle from the cache which is passed to the manager. the manager uses the bundle to prepopulate the cache. I wasn't clear on the repercussions or losing JWT keys there...

Reply:
I guess it would probably be ok... since workloads wouldn't be able to fetch svid's (and therefore bundles) anyway until the agent had contacted the server and refreshed bundles anyway.